### PR TITLE
Christine's Text changes - first commits 

### PR DESCRIPTION
**Pull request by christinerogers**
Tweaked language to make it less technical and more effective for expected users, e.g. : 
* folder instead of directory
* compressed package instead of tarball. 
* some emojis consolidated for effective use of colour to draw attention at key places 
* User-friendly prompts (e.g. Line frequency for `line_freq`)

To do:  @maltheism I could your advice here - let me know if I should tag you in specific places? 

* replace `href` instances in the commits and with proper links to given targets
* Version number - should this come from a Version file?  (always useful for bug reporting) 
and... testing after merge

### CODE REVIEW

**Comment by christinerogers** [comment]
Hi @maltheism , I've made the commits agreed in our last meeting (ones for me to do), and adjusted maybe 3 other truly tiny.  
Not quite sure I nailed the `&nbsp; `tweaks -- Could you please send me screenshots and let me know what looks wonky? 
This repo is now public, fyi. 

Thanks!  re-requesting review :)

I think still to be done is : 
* emoji changes agreed last week 
* versioning - did you want any input from me? (see you on slack)

File: src/jsx/Validator.js [hunk 1]
```diff
@@ -114,11 +114,11 @@ const Validator = (props) => {
       <div className={'info'}>
         <div className={'small-pad'}>
           <b style={{cursor: 'default'}}>
-            Confirm BIDS data structure:&nbsp;
+            Run BIDS Validator:&nbsp;
           </b>
           <input onClick={validateBIDS}
             type={'button'}
-            value={'Validate BIDS'}/>
+            value={' Validate BIDS '}/>
```
Comment (maltheism) [inline src/jsx/Validator.js:121]: Hi @christinerogers, the spaces need to be removed.

File: src/jsx/Welcome.js [hunk 2]
```diff
@@ -30,50 +30,59 @@ const Welcome = (props) => {
...
         </p>
       </div>
       <div>
         {/*<input value={'Settings'} type={'button'} onClick={openSettings}/>*/}
       </div>
       <div className={'footer'}>
         Powered by <a className={'open-source'} onClick={openGitHub}>
-        open source software</a>.<br/>
-        Copyright © 2021 MCIN.
+        open source software</a><br/>
```
Comment (maltheism) [inline src/jsx/Welcome.js:84]: Hi @christinerogers, the `.` need to be reverted.

File: src/jsx/Converter.js [hunk 3]
```diff
@@ -26,15 +26,14 @@ const Converter = (props) => {
   const [modalText, setModalText] = useState({
     mode: 'loading',
     title: {
-      loading: '⭐ Task Started!',
-      success: '🌈 Task Finished!',
-      error: '❌ Task Error!',
+      loading: ' Task in Progress',
+      success: ' Success - Complete',
```
Comment (maltheism) [inline src/jsx/Converter.js:30]: Hi @christinerogers, changes here need to be reverted.

**Comment by maltheism** [comment]
Hi @christinerogers, I noticed a few things that need to be reverted and I think it's causing Conflicting files as you see above cannot be merged until you rebase.

**Review by maltheism — CHANGES_REQUESTED** [review]

File: src/jsx/Validator.js (hunk 1 above)
Comment (christinerogers) [reply src/jsx/Validator.js:121]: done

File: src/jsx/Welcome.js (hunk 2 above)
Comment (christinerogers) [reply src/jsx/Welcome.js:84]: done

File: src/jsx/Converter.js (hunk 3 above)
Comment (christinerogers) [reply src/jsx/Converter.js:30]: ok line 30 reverted.

**Comment by christinerogers** [comment]
all comments resolved, just need to rebase and we'll be good to go.  It's not working through github so I'll dig in further.

**Pull request closed without merging**